src/dmd/attrib.d

```diff
@@ -451,6 +451,66 @@ extern (C++) final class CPPMangleDeclaration : AttribDeclaration
     }
 }
 
+/***********************************************************
+ */
```

Comment:
No documentation

Reply:
As much as the existing ones. But point taken, will add some documentation.